main/FatesRestartInterfaceMod.F90

```diff
@@ -2207,9 +2211,25 @@ subroutine set_restart_vectors(this,nc,nsites,sites)
              end do
           end do
 
+          do i_scls = 1, nlevsclass
+             do i_pft = 1, numpft
+                do i_term_type = 1, n_term_mort_types
+                   rio_termnindiv_cano_siscpf(io_idx_si_scpf_term) = sites(s)%term_nindivs_canopy(i_term_type,i_scls,i_pft)
+                   rio_termnindiv_usto_siscpf(io_idx_si_scpf_term) = sites(s)%term_nindivs_ustory(i_term_type,i_scls,i_pft)
+                   io_idx_si_scpf_term = io_idx_si_scpf_term + 1
+                end do
+             end do
+          end do
+
+          do i_pft = 1, numpft
+             do i_term_type = 1, n_term_mort_types
+             rio_termcflux_cano_sipft(io_idx_si_pft_term)  = sites(s)%term_carbonflux_canopy(i_term_type,i_pft)
+             rio_termcflux_usto_sipft(io_idx_si_pft_term)  = sites(s)%term_carbonflux_ustory(i_term_type,i_pft)
+             io_idx_si_pft_term = io_idx_si_pft_term + 1
+             end do
```

Comment:
I think this loop could also be nested in to the `i_pft` loop starting on line 2232 below.